ui/src/data-services/models/pipeline.ts

```diff
@@ -82,6 +83,22 @@ export class Pipeline {
     })
   }
 
+  get hasOnlineProcessingService(): {
```

Comment:
Nit: since this getter is returning an object (not a boolean value), I would suggest renaming it. Maybe just `currentProcessingService`?